### DISCUSSION TITLE
Post Processing 빌드에서 안돼요

### DISCUSSION THREAD

**Discussion by Deytooh** [naverz/zepeto-studio-kor#574] [Issue Report (Legacy)]
안녕하세요 ~ 

지난주 빌드 해도 Post Processing 모바일에서 잘 돼고있었지만 오늘 아침에 빌드하고 테스트 했는데 갑자기 안돼요 ㅠㅠ
Zepeto 업데이트 해서 그런가요 ?
World Version : 1.1.5
Zepeto App Version : 3.11.0
Mobile : Google Pixel 5 ( 아 그리고 Google Pixel 5에서 월드에서 IWP 하면 앱 crash 해요 ) 

이거 나와요 =>
![image](https://user-images.githubusercontent.com/97385930/161495274-cb0c2c27-8ca1-46bb-83b1-4a56e10b30aa.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 이슈 제보 감사합니다.
관련 현상에 대해 확인 및 수정 작업 진행 중에 있습니다. 🙏

**Reply by Deytooh** [reply]
@Zepeto-TechEvangelist 혹시 어떻게 돼고있어요 ? 😄

**Reply by Deytooh** [reply]
오 ! 돼네요 ! 감사합니다 :)

**Reply by Zepeto-TechEvangelist** [reply]
넵 최신 제페토 앱 업데이트에서 수정 되었습니다.
감사합니다 :)
